app/javascript/packs/globals.js

```diff
@@ -71,3 +71,10 @@ require('codemirror/mode/xml/xml.js');
 require('codemirror/mode/yaml/yaml.js');
 require('codemirror/lib/codemirror.css');
 require('codemirror/theme/eclipse.css');
+
+// bootstrap-datepicker language catalogs
+require('bootstrap-datepicker/dist/locales/bootstrap-datepicker.es.min.js');
+require('bootstrap-datepicker/dist/locales/bootstrap-datepicker.fr.min.js');
+require('bootstrap-datepicker/dist/locales/bootstrap-datepicker.ja.min.js');
+require('bootstrap-datepicker/dist/locales/bootstrap-datepicker.pt-BR.min.js');
+require('bootstrap-datepicker/dist/locales/bootstrap-datepicker.zh-CN.min.js');
```

Comment:
why just these particular languages?

Reply:
These are the languages that we currently make available in our application.

Reply:
we support more since Jansa - trying to get them forward-ported here: https://github.com/ManageIQ/manageiq/pull/20719

more importantly, though - the instructions [here](https://www.manageiq.org/docs/guides/i18n#manageiq-the-main-project) on adding a language are insufficient

Reply:
@chessbyte quite right, but right now I'm working on a solution so that no docs changes would be needed and things would be done automatically.

Reply:
@chessbyte the automatic webpack file generation is implemented here: https://github.com/ManageIQ/manageiq/pull/20769